LPTokenWrapper.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "./DraculaToken.sol";
+
+contract LPTokenWrapper {
```

Comment:
Not sure if this class is needed in general. Suggesting the transferring of the logic to the RewardPool.